src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static nature() {
+    this.alive = this.alive.filter((dead) =>
+      dead.health !== 0);
+  }
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
+
+  injure(healthPoints) {
+    this.health -= healthPoints;
+
+    if (this.health <= 0) {
+      Animal.alive.filter((animal) => animal.health <= 0);
```

Comment:
[CODE PERFORMANCE] The filter function inside the `injure` method is not removing the dead animals from the `Animal.alive` array, as it's not assigned back to `Animal.alive`. Moreover, the filter operation is redundant here because the `nature` method (which should be renamed) is already responsible for cleaning up dead animals. You should remove the filter operation from here.